### ISSUE TITLE
Smaller front knob for V0.2 which has the endstop at the bottom 

### ISSUE THREAD

**Issue by tkalbitz** [sryates/Snow_Knob#2]
First of all: I love the knobs. Thanks a lot for sharing :-) The V0.2 has the endstop at the bottom. The front knob is hitting the extrusion before the endstop is reached. Would it be possible to have a smaller for the front? I attached an image to show the collision.

![Visualization](https://user-images.githubusercontent.com/1212671/233807603-7f568b23-a051-431c-9ad7-8ab243cb761f.png)

**Comment by tkalbitz** [comment]
I [designed a knob](https://www.printables.com/model/460199-voron-v02-bed-leveling-knob) myself for the front. I don't know how you designed the outside gripping feature, but it's much better then mine :-)

**Comment by sryates** [comment]
I know what you are talking about on the collision I could make the front slightly smaller than the rest but if your endstop is moved just a MM up you won't have any issues with collision.  
The outside grip I stewed over them for a bit before using a combination of a lot of radius and such and several testing to see what felt good and what didn't I ended up on this after about 3-4 versions. Because the Zstop is located on the bottom I actually had more issues with my screws hitting the bottom not my knobs lol.
